internal/file/unicode/generator/gen_chars.go

```diff
@@ -0,0 +1,76 @@
+//+build gen
+
+package main
+
+//go:generate go run gen_chars.go
+
+// embeds chars.txt which itself is a copy of https://raw.githubusercontent.com/codebox/homoglyph/master/raw_data/chars.txt
+// make sure you update that file and run "make gen" from the root of the project
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+)
+
+const (
+	charsFile = "chars.txt"
+	charKeys  = ".-_aejmnprwv" // maven-wrapper.jar and _
+	target    = "../chars.go"
+)
+
+func main() {
```

Comment:
I think this logic might fail if there's multiple letters mapping to the same letters in `maven-wrapper.jar` and the map wont be fully populated. Might have to rework this a wee bit.

Reply:
fixed